src/components/AvatarGroup/components/SbMoreAvatars.vue

```diff
@@ -81,9 +81,14 @@ export default {
       }
     },
 
+    closeAvatarsList() {
+      this.expanded = false
+    },
+
     handleClick(event) {
       this.toggleExpanded()
-      this.$emit('click', event)
+
+      this.$emit('click', event, this)
```

Comment:
can't you access this through a `ref` or the props that are provided to the component? Why is it necessary to emit the whole vm?

Reply:
also event emitting doesn't work with a third parameter no? Aren't you supposed to pass multiple parameters through an object?

```
this.$emit('success', {
  username: 'CodyBontecou',
  error: false,
})
```